.rubocop.yml

```diff
@@ -1,3 +1,10 @@
 # use the shared Yast defaults
 inherit_from:
   /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+
+Metrics/LineLength:
+  Max: 150
```

Comment:
You could do something like this to avoid that problem:

<pre>
CREATE_SNAPSHOT_CMD = "/usr/bin/snapper create --type=%{snapshot_type} " \
  "--cleanup-algorithm=number --print-number --description=\"%{description}\""
</pre>